From 6e0cd803b9d423f5b8d76270c4a6baa8948cba27 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Fri, 20 Sep 2019 14:02:15 +0300
Subject: [PATCH 1/1] Fix compilation under Vala 0.45 and above.

In 0.45+, creation methods for abstract classes can no longer be public. Change to protected to resolve issue.
---
 src/recording/base-screen-recorder.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/recording/base-screen-recorder.vala b/src/recording/base-screen-recorder.vala
index 04fec42..d3d7e1c 100644
--- a/src/recording/base-screen-recorder.vala
+++ b/src/recording/base-screen-recorder.vala
@@ -40,7 +40,7 @@ namespace Peek.Recording {
       }
     }
 
-    public BaseScreenRecorder () {
+    protected BaseScreenRecorder () {
       config = new RecordingConfig ();
     }
 
-- 
2.23.0

